pkg/version/check_test.go

```diff
@@ -341,7 +355,35 @@ func TestCheckPDVersionError(t *testing.T) {
 	resp = func(w http.ResponseWriter, _ *http.Request) {
 		w.WriteHeader(http.StatusInternalServerError)
 	}
-	require.Contains(t, CheckPDVersion(context.TODO(), ts.URL, nil).Error(),
+	require.Contains(t, checkPDVersion(context.TODO(), ts.URL, nil).Error(),
 		"[CDC:ErrCheckClusterVersionFromPD]failed to request PD 500 Internal Server Error , please try again later",
 	)
 }
+
+func TestCheckTiCDCVersion(t *testing.T) {
+	t.Parallel()
```

Comment:
I think we can use a map to organize the test cases.

For example:

map[case-name]struct{}